.github/CODEOWNERS

```diff
@@ -124,9 +124,10 @@
 /upsc/                                   @platinummonkey
 /uptime/                                 @jeremy-lq
 /vercel/                                 @Datadog/serverless
-/vespa/                                  @gjoranv dd@vespa.ai
```

Comment:
Looks like this is throwing an error - did you purposefully remove this email from the codeowners file?

Reply:
I added that email addr back in